lib/susedistribution.pm

```diff
@@ -428,7 +428,7 @@ sub init_consoles {
 
     if (is_qemu) {
         $self->add_console('root-virtio-terminal', 'virtio-terminal', {});
-        $self->add_console('user-virtio-terminal', 'virtio-terminal', {});
+        $self->add_console('user-virtio-terminal', 'virtio-terminal', {socked_path => cwd() . '/virtio_console_user'});
```

Comment:
@cfconrad @asmorodskyi @Martchus @dzedro @foursixnine  IMHO the API is bad. Socket path in `consoles::virtio_terminal->socked_path` should be determined from the fact if it's user console or root console and from id (we have more root consoles), see: https://github.com/os-autoinst/os-autoinst/blob/b66759aea2b005800a00ddf5f8a405fc3c8b496b/consoles/virtio_terminal.pm#L42-L46

I.e. instead having here new class attributes `root` (`1` => root console, `0` => user console) and `id`.

That way the socket name wouldn't be on 2 places on os-autoinst and here, but just in os-autoinst (2 places: the other is in qemu.pm: https://github.com/os-autoinst/os-autoinst/blob/b66759aea2b005800a00ddf5f8a405fc3c8b496b/backend/qemu.pm#L487-L490).